src/main/java/io/github/onecx/product/store/bff/rs/log/ProductStoreLog.java

```diff
@@ -0,0 +1,52 @@
+package io.github.onecx.product.store.bff.rs.log;
+
+import java.util.List;
+
+import jakarta.enterprise.context.ApplicationScoped;
+
+import org.tkit.quarkus.log.cdi.LogParam;
+
+import gen.io.github.onecx.product.store.bff.rs.internal.model.*;
+
+@ApplicationScoped
+public class ProductStoreLog implements LogParam {
+
+    @Override
+    public List<LogParam.Item> getClasses() {
+
+        return List.of(
+                this.item(10, CreateMicrofrontendRequestDTO.class,
+                        x -> "CreateMicrofrontendRequestDTO[ appId: " +
+                                ((CreateMicrofrontendRequestDTO) x).getAppId()
+                                + ", AppName: " + ((CreateMicrofrontendRequestDTO) x).getAppName()
+                                + ", remoteBaseUrl: " + ((CreateMicrofrontendRequestDTO) x).getRemoteBaseUrl()
+                                + ", remoteEntry: " + ((CreateMicrofrontendRequestDTO) x).getRemoteEntry()
+                                + ", productName: " + ((CreateMicrofrontendRequestDTO) x).getProductName()
```

Comment:
Andrej advised me sometime ago to just use one value for each log-param

Reply:
That's right - focus on BIDs or any other unique field which is necessary to identify within backend system - no more. And avoid whitespace as well.